### ISSUE TITLE
Switch Component Renders Differently on RNW vs Native

### ISSUE THREAD

**Issue by hirvesh** [tamagui/tamagui#3072]
### Current Behavior

The switch component with the default Tamagui theme looks different on native vs web. On native, when the switch is moved to "on" state, color looks a muted gray (could not change it via themes either) and on web it looks black in the "on" state.

See video for what I mean.

https://github.com/user-attachments/assets/51f5fbc9-84a7-4398-9b57-5863080c3afa

### Expected Behavior

I am expecting the switch on native to follow the same colors as on web as currently on native, switch looks "disabled" in "on" state.

### Tamagui Version

```markdown
1.116.14
```

### Platform (Web, iOS, Android)

```markdown
Web, iOS, Android
```

### Reproduction

```markdown
Any new Tamagui project will give the same result from the starter template.
```

**Comment by iameli** [comment]
Yep, seeing this as well. Is Switch just broken by default? Why does the one in the docs not look like this?
